jax/_src/numpy/ufuncs.py

```diff
@@ -1984,9 +1984,41 @@ def frexp(x: ArrayLike, /) -> tuple[Array, Array]:
   return _where(cond, x, x1), lax.convert_element_type(x2, np.int32)
 
 
-@implements(np.remainder, module='numpy')
 @jit
 def remainder(x1: ArrayLike, x2: ArrayLike, /) -> Array:
+  """Returns element-wise remainder of the division.
+
+  JAX implementation of :obj:`numpy.remainder`.
+
+  The remainder is computed by ``x1 - x2 * jnp.floor(x1 / x2)``.
+
+  Args:
+    x1: scalar or array. Specifies the dividend.
+    x2: scalar or array. Specifies the divisor. ``x1`` and ``x2`` should either
+      have same shape or be broadcast compatible.
+
+  Returns:
+    An array containing the remainder of element-wise division of ``x1`` by
+    ``x2`` with same dtype as inputs and sign as that of the elements of ``x2``.
```

Comment:
"same dtype as inputs" is not correct here: the output dtype is the type promotion result of `x1` and `x2` upcast to inexact.

Reply:
Thanks! Updated as suggested